specification/managementgroups/ManagementGroups.Management/HierarchySettings.tsp

```diff
@@ -0,0 +1,83 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/rest";
+import "./models.tsp";
+import "./ManagementGroup.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+
+namespace Microsoft.Management;
+/**
+ * Settings defined at the Management Group scope.
+ */
+@singleton("default")
+@parentResource(ManagementGroup)
+model HierarchySettings
+  is Azure.ResourceManager.ProxyResource<HierarchySettingsProperties> {
+  ...ResourceNameParameter<
+    Resource = HierarchySettings,
+    KeyName = "setting",
+    SegmentName = "settings",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface HierarchySettingsOperationGroup {
+  /**
+   * Gets the hierarchy settings defined at the Management Group level. Settings can only be set on the root Management Group of the hierarchy.
+   *
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "non-standard operations"
+  get is Extension.Read<Extension.Tenant, HierarchySettings>;
```

Comment:
I think we should just use the ordinary `ArmResourceRead` template here for ordinary resources.